tests/tracer/test_single_span_sampling_rules.py

```diff
@@ -119,6 +122,56 @@ def test_single_span_rules_do_not_tag_if_tracer_samples():
         assert_sampling_decision_tags(span, sample_rate=None, mechanism=None, limit=None, trace_sampling=True)
 
 
+def test_sampling_rule_init_config_multiple_sampling_rule_json_via_file(tmpdir):
+    file = tmpdir.join("rules.json")
+    file.write(
+        '[{"service":"xy?","name":"a*c"}, \
+            {"sample_rate":0.5,"service":"my-service","name":"my-name", "max_per_second":"20"}]'
+    )
+
+    with override_env(dict(DD_SPAN_SAMPLING_RULES_FILE=str(file))):
+        sampling_rules = get_file_json()
```

Comment:
Do we want to be calling `get_span_sampling_rules` instead?

Reply:
Given prior feedback about breaking the function up a bit more, I thought it made sense just to test that get_file_json() returned what we expected.